### DISCUSSION TITLE
How to customize triangles preset to div repulse

### DISCUSSION THREAD

**Discussion by ashishshaiju** [matteobruni/tsparticles#5201] [Q&A]
I mean how to **add div repulse** to **triangles preset.**
div-repulse: https://particles.js.org/samples/#divRepulse
Trinagles: https://particles.js.org/samples/presets/triangles.html

**Comment by matteobruni** [comment]
You can override preset options using the normal config like this:

```
{
  preset: "triangles",
  particles: {
    number: {
      value: 10
    }
  }
}
```

If you want to add div repulse, override the interactivity section in your config
